hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
 rrhs:
-  - ttl: 600
-    type: CNAME
-    value: rrhs-hack-club.netlify.app.
+- ttl: 300
+  type: CNAME
+  value: rrhs-hack-club.netlify.app.
 ru:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 runshaw:
-  - ttl: 600
-    type: CNAME
-    value: dragon863.hackclub.app.
+- ttl: 300
+  type: CNAME
+  value: dragon863.hackclub.app.
 s1._domainkey:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: s1.domainkey.u9906371.wl122.sendgrid.net.
 s2._domainkey:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: s2.domainkey.u9906371.wl122.sendgrid.net.
 sa:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: hc-sa.netlify.app.
 saleh:
-  - ttl: 600
-    type: NS
-    values:
-      - ns1.he.net.
-      - ns2.he.net.
-      - ns3.he.net.
-      - ns4.he.net.
-      - ns5.he.net.
+- ttl: 300
+  type: NS
+  values:
+  - ns1.he.net.
+  - ns2.he.net.
+  - ns3.he.net.
+  - ns4.he.net.
+  - ns5.he.net.
 santa:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: ancient-pinchusion-nm14orueibm39fhm2ntl24qj.herokudns.com.
 saycheese:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 schedule:
-  - type: CNAME
-    value: calendso-production-f2a4.up.railway.app.
+- ttl: 300
+  type: CNAME
+  value: calendso-production-f2a4.up.railway.app.
 schedule.webdev:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 scholarship:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: scholarship-hackclub.netlify.com.
 school-toolbox:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 school-toolkit:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 scrapbook:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 scrapbook.assemble:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 scrapbooks:
-  - ttl: 600
-    type: CNAME
-    value: keen-brahmagupta-b1f934.netlify.app.
+- ttl: 300
+  type: CNAME
+  value: keen-brahmagupta-b1f934.netlify.app.
 scrappy:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: a.selfhosted.hackclub.com.
 scrapyard:
-  - ttl: 60
-    type: TXT
-    values:
-      - v=spf1 include:_spf.google.com ~all
-      - google-site-verification=yZxQNIGF8LvRVUJ_LpekD1PTKCTFj2ECQZBA0OZXQus.
-      - v=DMARC1\; p=none\; rua=mailto:dmarc_report@hackclub.com\; ruf=mailto:dmarc_report@hackclub.com\;fo=1
-  - ttl: 60
-    type: MX
-    values:
-      - exchange: aspmx.l.google.com.
-        preference: 1
-      - exchange: alt1.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt2.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt3.aspmx.l.google.com.
-        preference: 10
-      - exchange: alt4.aspmx.l.google.com.
-        preference: 10
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?